src/main/resources/flyway/sql/V2__new_table_primarysite_cancertype.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TYPE membership AS ENUM ('FULL', 'ASSOCIATE');
+
+CREATE TABLE cancer (
+    id                UUID PRIMARY KEY,
+    name              VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE site  (
```

Comment:
Please change this to `primary_site`, as its more clear. Likewise, update the wiki
https://wiki.oicr.on.ca/pages/viewpage.action?spaceKey=icgcargotech&title=Program+Service+-+Backend